packages/core-components/src/components/AlertDisplay/AlertDisplay.tsx

```diff
@@ -83,7 +83,7 @@ export function AlertDisplay(props: AlertDisplayProps) {
         severity={firstMessage.severity}
       >
         <span>
-          {firstMessage.message.toString()}
+          {`${firstMessage?.message}`}
```

Comment:
```suggestion
          {String(firstMessage.message)}
```

I prefer the intent of this, but that's entirely optional.

We'll still crash on undefined messages btw, so suggested a change for that too. Feeling it's far into the category of stuff that shouldn't happen and that we want to know about.

Reply:
Also, the `firstMessage.severity` without a question mark on line 83 still would lead to a crash if we allowed firstMessage to be null/undefined.

Reply:
Yeah, preferably we'd somehow deal with this upstream, from the Alert/Error forwarder (where one could `new Error().stack` for reference if the message isn't valid), or in an observable `.map` step (though not as traceable)...